### PR TITLE
Fix a typo and update one url

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
Fix a typo and update one url

> NOTE: Please check the “Allow edits from maintainers” box below to allow 
> reviewers fix problems on your patch and speed up the review process.
> Please delete this note before submitting the pull request.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3041)

### CODE REVIEW

File: docs/concepts/cluster-administration/authenticate-across-clusters-kubeconfig.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Multiple kubeconfig files are allowed, if specified explicitly.  At runtime they
 
 ## Related discussion
 
-http://issue.k8s.io/1755
```
Comment (mikedanese) [inline docs/concepts/cluster-administration/authenticate-across-clusters-kubeconfig.md]: This is correct as is.

**Comment by mikedanese** [comment]
@chenopis why did you merge this when I had an outstanding comment?

**Comment by chenopis** [comment]
@mikedanese Sorry. http://issue.k8s.io/1755 redirects to https://github.com/kubernetes/kubernetes/issues/1755 so I thought using the final URL was fine. Is it better to use the issue.k8s.io format?

**Comment by chenopis** [comment]
Since the compact form, http://issue.k8s.io/1755, makes things simpler, I'm reverting to that in PR #3047 and turning it into a clickable link.

**Pull request merged**